.gitignore

```diff
@@ -0,0 +1,123 @@
+### IntelliJ project files
```

Comment:
I don't think that gitignore is needed in this repo – there will be only one file here – README.md

Reply:
Removed :)